internal/cmd/plugins.go

```diff
@@ -28,7 +28,8 @@ func plugs() plugins.List {
 func decorate(name string, cmd *cobra.Command) {
 	pugs := plugs()
 	for _, c := range pugs[name] {
-		anywhereCommands = append(anywhereCommands, c.Name)
```

Comment:
plugins are not anywhere commands. they should be executed within the app directory.